v3/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ project(LGR VERSION 3.0.0 LANGUAGES CXX)
 
 option(LGR_ENABLE_CUDA "Build GPU support" OFF)
 option(LGR_ENABLE_UNIT_TESTS "Enable unit tests" ON)
+option(LGR_ENABLE_EFENCE "Build with ElectricFence support" OFF)
```

Comment:
Have you tried Clang's built-in address sanitizer?

Reply:
No! Need to learn hot to use that!